src/Service/MicrosoftGraphBookingService.php

```diff
@@ -508,6 +509,23 @@ public function getUserBookingFromApiData(array $data): UserBooking
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getAllFutureBookings($token): GraphResponse
```

Comment:
Should return an array instead of a graph response.
For example, see https://github.com/itk-dev/book_aarhus/blob/develop/src/Service/MicrosoftGraphBookingService.php#L354

Reply:
Fixed